app/_hub/nonamesecurity/nonamesecurity-kongtrafficsource/overview/_index.md

```diff
@@ -16,6 +16,11 @@ For example, an action could be fetching a log file, or to initiating a block ba
 The Noname Security Kong plugin is available as a LuaRocks module.
 The [Noname Security install and configuration documentation](https://docs.nonamesecurity.com/docs/kong-plugin) explains how to log on to the Noname admin user interface, go to the Integrations section of the platform, begin the Kong integration wizard, and download the plugin that is used in the Dockerfile to create a custom docker image with the plugins preinstalled. 
 
+Prevention is enabled by default. To disable the prevention feature, see https://docs.nonamesecurity.com/docs/kong-plugin#disabling-the-prevention-feature 
```

Comment:
wrap the url in the markdown link format so that the link highlights and is clickable